### PR TITLE
Align with existing transformations

### PR DESCRIPTION
**Pull request by jamesmkrieger**
This adds another option for using the alignment protocol

### CODE REVIEW

File: prody2/protocols/protocol_atoms.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
 from pyworkflow.utils import *
 from pyworkflow.protocol.params import (PointerParam, StringParam, FloatParam,
-                                        EnumParam, LEVEL_ADVANCED)
+                                        BooleanParam, EnumParam, LEVEL_ADVANCED)
 
 import prody
 from prody import LOGGER
```
Comment (pconesa) [inline prody2/protocols/protocol_atoms.py:45]: Should we go for the standard way of logging

**Review by pconesa — CHANGES_REQUESTED** [review]
Use standard logging methods?

File: prody2/protocols/protocol_atoms.py (hunk 1 above)
Comment (jamesmkrieger) [reply prody2/protocols/protocol_atoms.py:45]: Yes. We have some in #34 and I'll do the same for the new things here.

**Comment by jamesmkrieger** [comment]
The changes here do not introduce new logging. All the logging changes we need are in #34

**Pull request merged**